pkgs/by-name/ff/ffts/package.nix

```diff
@@ -0,0 +1,42 @@
+{
+  stdenv,
+  lib,
+  cmake,
+  git,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "ffts";
+  version = "0.7";
+
+  src = fetchFromGitHub {
+    owner = "ValveSoftware";
+    repo = "ffts";
+    rev = "2c8da4877588e288ff4cd550f14bec2dc7bf668c";
+    hash = "sha256-Cj0n7fwFAu6+3ojgczL0Unobdx/XzGNFvNVMXdyHXE4=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    git
+  ];
+  buildInputs = [ ];
+
+  cmakeFlags = [
+    "-DCURRENT_GIT_VERSION=${lib.substring 0 7 src.rev}"
+    "-DENABLE_SHARED=ON"
+    "-Wno-deprecated"
+  ];
+
+  meta = with lib; {
+    description = "The Fastest Fourier Transform in the South";
+    homepage = "http://anthonix.com/ffts";
+    license = licenses.mit;
+    maintainers = with lib.maintainers; [
```

Comment:
Why are you not putting `bgamari` as maintainer?
Does `thoughtpolice` and `hansfbaier` want to maintain this package?

Reply:
This came from the original MR, https://github.com/NixOS/nixpkgs/pull/29816. I'll let @hansfbaier and @thoughtpolice confirm.

I will add myself as well.